PocketKit/Sources/PocketKit/Article/Item+Readable.swift

```diff
@@ -1,6 +1,16 @@
 import Sync
+import Foundation
+
+
+extension SavedItem: Readable {
+    var readerURL: URL? {
+        item?.resolvedURL ?? item?.givenURL ?? url
```

Comment:
Looks like this mimics `SavedItem.bestURL`. Potentially a place for piggybacking off of it.